tests/src/Functional/ContentEventRenderTest.php

```diff
@@ -24,6 +24,7 @@ class ContentEventRenderTest extends WhitelabelBrowserTestBase {
    */
   public static $modules = [
     'block',
+    'oe_corporate_blocks',
```

Comment:
This is a content type test, and we need to manually enable corporate blocks. This is exactly what we should try to avoid and fix. Please revert.